### PR TITLE
GUI: Add feature to copy text from console

### PR DESCRIPTION
**Pull request by Darkhood148**
## Add Feature: Copy text from console
image

### CODE REVIEW

**Comment by sev-** [comment]
Thank you! Please follow our commit guidelines, code formatting, indent. Then we will do a deeper review

File: gui/widgets/edittext.cpp [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
```
Comment (bluegr) [inline gui/widgets/edittext.cpp]: Why? Please revert this

**Comment by bluegr** [comment]
There are a lot of unneeded whitespace changes (e.g. in edittext.cpp).

Please check our [Code formatting guidelines](https://wiki.scummvm.org/index.php?title=Code_Formatting_Conventions), and adjust accordingly - you will probably need to adjust your IDE's whitespace settings - check our [`.editorconfig`](https://github.com/scummvm/scummvm/blob/master/.editorconfig) file

**Comment by sev-** [comment]
Thank you! All works great now!

**Pull request merged**